src/generic/modal-dropzone/ModalDropzone.jsx

```diff
@@ -129,7 +129,7 @@ ModalDropzone.defaultProps = {
   imageHelpText: '',
   previewComponent: null,
   imageDropzoneText: '',
-  maxSize: UPLOAD_FILE_MAX_SIZE,
+  maxSize: '',
```

Comment:
```suggestion
  maxSize: undefined,
```
Sine `maxSize` is an optional number, we want `undefined` instead of `''` to indicate unspecified max size here.